cylc/uiserver/scripts/gui.py

```diff
@@ -51,8 +55,32 @@
 
 def main(*argv):
     init_log()
+    hub_url = glbl_cfg().get(['hub', 'url'])
     jp_server_opts, new_gui, workflow_id = parse_args_opts()
+    if '--help' in sys.argv and hub_url:
+        print(
+            dedent('''
+                cylc gui [WORKFLOW]
+
+                Open the Cylc GUI in a new web browser tab.
+
+                If WORKFLOW is specified, the GUI will open on this workflow.
+
+                This command has been configured to use a centrally configured
+                Jupyter Hub instance rather than start a standalone server.
+                To see the configuration options for the server run
+                "cylc gui --help-all", these options can be configured in the
+                Jupyter configuration files using "c.Spawner.cmd", see the Cylc
+                and Jupyter Hub documentation for more details.
+            '''))
+        return
     if '--help' not in sys.argv:
```

Comment:
This will let the `--help-all` option through:

```suggestion
    if not {'--help', '--help-all'} & set(sys.argv):
```